### PR TITLE
SDIT-1513 Add pageSize for housekeeping to allow max number of records when retrying exception queue

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/integration/OracleToTopicIntTest.kt [hunk 1]
```diff
@@ -153,6 +153,32 @@ class OracleToTopicIntTest : IntegrationTestBase() {
...
+        .expectStatus().isOk
+
+      awaitExceptionQueueSizeToBe(1)
+      awaitQueueSizeToBe(1)
+
+      webTestClient
+        .put().uri("/housekeeping?pageSize=1")
+        .headers(setAuthorisation(roles = emptyList()))
+        .exchange()
+        .expectStatus().isOk
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/integration/OracleToTopicIntTest.kt:177]: How are we going to call this several times until the DLQ is empty?

Comment (karenmillermoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/integration/OracleToTopicIntTest.kt:177]: line 179 checks the dlq is empty

Comment (mikehalmamoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/integration/OracleToTopicIntTest.kt:177]: Sorry I mean in real life - the housekeeping endpoint only gets called once every 10 minutes.

**Pull request merged**